torch/csrc/jit/script/compiler.cpp

```diff
@@ -1657,6 +1657,38 @@ struct to_ir {
     }
   }
 
+  void emitAugAssignToContainer(
+      const std::string& containerType,
```

Comment:
Passing the container type as a list is very suspicious. Can we pass the actual `Type` and use `python_str()` methods and such for reporting errors?